<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisWindow</class>
 <widget class="QMainWindow" name="AnalysisWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Waterfall Analysis</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolBar#toolBar {
background: red;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
padding: 0;
}

QWidget#parameters_widget {
background-color: rgb(247, 239, 214);
padding: 0;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="parameters_widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSlider" name="slider_angle">
         <property name="maximum">
          <number>359</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_angle">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Background</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_bkg_axis">
         <property name="toolTip">
          <string>Background axis</string>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_bkg">
         <property name="toolTip">
          <string>Background sigma</string>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::VisualMoveStyle</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="plot_widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open"/>
   </widget>
   <widget class="QMenu" name="menuWaterfall">
    <property name="title">
     <string>Waterfall</string>
    </property>
    <addaction name="action_transpose"/>
    <addaction name="action_background"/>
    <addaction name="action_setup_roi"/>
    <addaction name="action_apply_roi"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWaterfall"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover {background-color:lightgray}
QToolButton:pressed {background-color:darkgray} 
</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="action_transpose"/>
   <addaction name="action_crop"/>
   <addaction name="action_setup_roi"/>
   <addaction name="action_apply_roi"/>
   <addaction name="action_background"/>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset>
     <normaloff>Icons/folder-open-outline.svg</normaloff>Icons/folder-open-outline.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open waterfall</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_transpose">
   <property name="icon">
    <iconset>
     <normaloff>Icons/blueprint_shuffle.svg</normaloff>Icons/blueprint_shuffle.svg</iconset>
   </property>
   <property name="text">
    <string>Transpose</string>
   </property>
   <property name="toolTip">
    <string>Transpose waterfall</string>
   </property>
  </action>
  <action name="action_setup_roi">
   <property name="icon">
    <iconset>
     <normaloff>Icons/blueprint_guide.svg</normaloff>Icons/blueprint_guide.svg</iconset>
   </property>
   <property name="text">
    <string>Setup ROI</string>
   </property>
   <property name="toolTip">
    <string>Setup or clear ROI line</string>
   </property>
  </action>
  <action name="action_apply_roi">
   <property name="icon">
    <iconset>
     <normaloff>Icons/blueprint_live.svg</normaloff>Icons/blueprint_live.svg</iconset>
   </property>
   <property name="text">
    <string>Apply ROI</string>
   </property>
   <property name="toolTip">
    <string>Show data iniside ROI</string>
   </property>
  </action>
  <action name="action_background">
   <property name="icon">
    <iconset>
     <normaloff>Icons/bent-blocks_erase.svg</normaloff>Icons/bent-blocks_erase.svg</iconset>
   </property>
   <property name="text">
    <string>Subtract Background</string>
   </property>
   <property name="toolTip">
    <string>Remove Background</string>
   </property>
  </action>
  <action name="action_crop">
   <property name="icon">
    <iconset>
     <normaloff>Icons/bent-blocks_crop.svg</normaloff>Icons/bent-blocks_crop.svg</iconset>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
   <property name="toolTip">
    <string>Select interesting range</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
